### ISSUE TITLE
Window's iframe renders without title attribute

### ISSUE THREAD

**Issue by jivanova** [telerik/kendo-ui-core#4879]
### Bug report
[DOJO](https://dojo.telerik.com/eqiliQew/2)

The issue is reproducible with `iframe:true` configuration. The best approach for the matter would be to allow title configuration as:
```
iframe: {
    title: "Test"
}
```

### Reproduction of the problem
1. Run Accessibility scan

### Current behavior
Title is empty error - https://dequeuniversity.com/rules/axe/2.2/frame-title

### Expected/desired behavior
The iframe should be rendered with title

### Environment
* **Kendo UI version:** 2019.1.220
* **Browser:** all

**Comment by veselints** [comment]
The `<iframe>` title value will be taken from the `title` configuration option of the Window. Therefore, the following setup will properly render `title` attribute value for the `<iframe>`:
```js
$("#dialog").kendoWindow({
  content: "http://www.telerik.com/",
  iframe: true,
  title: "window title"
});
```

Here is a Dojo sample implementing the above:
https://dojo.telerik.com/eqiliQew/8
